src/shared/components/events-list/index.js

```diff
@@ -2,134 +2,123 @@
+                <li key={`event_${event.id}`} className={styles.eventItem}>
+                  <Grid fit={false}>
+                    <Cell size={12}>
+                      <HR color="grey" customClassName=
+                        {styles.mobileHorizontalLine} />
+                      <DateBubble
+                          startDateTime={event.startDateTime}
+                          endDateTime={event.endDateTime}
+                          displayDateRange={(timeline === 'today' &&
+                            event.startDateTime.date !==
+                              event.endDateTime.date)}
+                      />
+                    </Cell>
+                    <Cell size={1} key="event_picture_mobile" hideOn="mobileSM">
+                      <EventImage imgPath={ imageAssetsEndpoint +
+                        (event.featureImageFilename ?
+                          event.featureImageFilename :
+                          'red-badger-event.jpg') } href={eventHref(event)} />
+                    </Cell>
+                    <Cell size={12} breakOn="mobile">
+                      <Grid fit={false}>
+                        <Cell size={8} key='event_description'
+                          breakOn="mobileS">
+                          <EventTitle eventTitle={event.title}
+                            eventHref={eventHref(event)} />
+                          <div className={styles.eventDescription}>
+                            {event.strapline}
+                          </div>
+                          {
+                            event.externalLinks || event.internalLinks ?
+                              <div className={styles.eventLinks}>
+                              {
+                                event.externalLinks ?
+                                  <EventLinksList
+                                    linkList={event.externalLinks}
+                                    listType="external" />
+                                  : null
+                              }
+                              {
+                                event.internalLinks ?
+                                  <EventLinksList
+                                    linkList={event.internalLinks}
+                                    listType="internal" />
+                                  : null
+                              }
+                              </div>
+                            : null
+                          }
+                          {
+                            event.tags
+                            ? <TagsList
+                                tags={event.tags}
+                                tagsLinkPath="about-us/events" />
+                            : null
+                          }
+                        </Cell>
+                        <Cell size={4} key='event_picture' breakOn="mobileS"
+                          hideOn="mobileS">
+                          <EventImage imgPath={ imageAssetsEndpoint +
```

Comment:
We do this in more than one place, could we extract it into a function? Also would pull logic out of the rendering bit :)